src/kili/llm/services/export/dynamic.py

```diff
@@ -60,25 +61,28 @@ def export(
                         label["id"],
                         obfuscated_models,
                     )
+                    formatted_response = _format_json_response(
+                        json_interface["jobs"],
+                        round["annotations"],
+                        round["completion"],
+                        obfuscated_models,
+                    )
+                    label_data = {
+                        "author": label["author"]["email"],
+                        "created_at": label["createdAt"],
+                        "label_type": label["labelType"],
+                        "label": formatted_response["label"],
+                    }
+                    if step == total_rounds - 1 and formatted_response["conversation_label"]:
+                        label_data["conversation_label"] = formatted_response["conversation_label"]
```

Comment:
I discussed with Stéphanie about that,  if we want to differentiate the type of the annotation at the label level, we need to do it for every levels ; for example : 

```
...
    "label": {
            "completion": {
                "RANKING": [
                    "B_2"
                ]
            },
            "conversation": {
                "TRANSCRIPTION_1": "some transcription at asset level"
            },
            "chat_item": {
                "012399314932R4": {
                    "TRANSCRIPTION_2": "some transcription at chat item level"
                }
            }
        }
```

Reply:
I'm not really ok with this format, as we duplicate some information already present in the json_interface. We can discuss it with the team if you want

Reply:
And doing this change mean applying it also on static export for consistency,  creating a breaking change in our export format

Reply:
No problem but could you tell me what format you have in mind ?